addons/mrp/models/mrp_production.py

```diff
@@ -1202,7 +1202,7 @@ def _set_qty_producing(self, pick_manual_consumption_moves=True):
                 continue
 
             # sudo needed for portal users
-            if move.sudo()._should_bypass_set_qty_producing():
+            if move in self.move_raw_ids and move.sudo()._should_bypass_set_qty_producing():
```

Comment:
because it was affecting the quantity changing of the byproducts. Before it, it was like impossible to update the quantity produced for a byproduct because the `_should_bypass_set_qty_producing` was `true`.